plugins/autoloader/prism-autoloader.js

```diff
@@ -170,7 +170,28 @@
 	}
 	var config = Prism.plugins.autoloader = {
 		languages_path: languages_path,
-		use_minified: true
+		use_minified: true,
+		/**
+		 * Loads the given language(s) and calls the given callback(s) asynchronously.
+		 *
+		 * The error callback will be called once for each language failing to load.
+		 *
+		 * @param {string|string[]} languages
+		 * @param {(languages: string[]) => void} [success]
+		 * @param {(language: string) => void} [error]
+		 */
+		loadLanguages: function (languages, success, error) {
+			// loadLanguages might call the callbacks synchronously, so we guarantee that this is not the case
```

Comment:
Maybe also put this comment into the&nbsp;description of&nbsp;the&nbsp;internal `loadLanguages(…)` function JSDoc.